deltacat/storage/model/manifest.py

```diff
@@ -358,6 +389,16 @@ def from_s3_obj_url(
         manifest_entry = ManifestEntry.of(url, manifest_entry_meta)
         return manifest_entry
 
+    @staticmethod
+    def from_dict(obj: dict) -> ManifestEntry:
+        return ManifestEntry.of(
+            url=obj.get("url"),
+            uri=obj.get("uri"),
+            meta=ManifestMeta.from_dict(obj.get("meta", {})),
```

Comment:
This behavior isn't fully consistent with the default constructor behavior of omitting the `meta` key completely if it doesn't exist (this will instead set the `meta` key to an empty dictionary).

```suggestion
            meta=ManifestMeta.from_dict(obj["meta"]) if "meta" in obj else None,
```

Reply:
addressed